proto/zitadel/user/v2/user_service.proto

```diff
@@ -2076,3 +2149,67 @@ enum AuthenticationMethodType {
   AUTHENTICATION_METHOD_TYPE_OTP_SMS = 6;
   AUTHENTICATION_METHOD_TYPE_OTP_EMAIL = 7;
 }
+
+message CreateInviteCodeRequest {
+  string user_id = 1 [
+    (validate.rules).string = {min_len: 1, max_len: 200},
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      min_length: 1;
+      max_length: 200;
+      example: "\"69629026806489455\"";
+    }
+  ];
+  // if no verification is specified, an email is sent with the default url and application name (ZITADEL)
+  oneof verification {
+    SendInviteCode send_code = 2;
+    ReturnInviteCode return_code = 3;
+  }
+}
+
+message CreateInviteCodeResponse {
+  zitadel.object.v2.Details details = 1;
+  // The invite code is returned if the verification was set to return_code.
+  optional string invite_code = 2;
+}
+message ResendInviteCodeRequest {
+  string user_id = 1 [
+    (validate.rules).string = {min_len: 1, max_len: 200},
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      min_length: 1;
+      max_length: 200;
+      example: "\"69629026806489455\"";
+    }
+  ];
+}
+
+message ResendInviteCodeResponse {
+  zitadel.object.v2.Details details = 1;
+}
+
+message VerifyInviteCodeRequest {
+  string user_id = 1 [
+    (validate.rules).string = {min_len: 1, max_len: 200},
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      min_length: 1;
+      max_length: 200;
+      example: "\"69629026806489455\"";
+    }
+  ];
+  string verification_code = 2 [
+    (validate.rules).string = {min_len: 1, max_len: 20},
```

Comment:
Do we have a check that the code can't be set at more than 20 characters in the config?